### ISSUE TITLE
Fails on conjoined verbs

### ISSUE THREAD

**Issue by genelkim** [genelkim/ulf-sanity-checker#25]
Possibly failed conditions:
  _"Verbs (both tensed and untensed) *must* take all non-subject arguments in a flat construction."_
Ann segment:
  ```
(((PASV FOLLOW.V)
  (BY.P-ARG
   (NP+PREDS ((THREE.D FOUR.D OR.CC SIX.D) (PLUR CHILD.N))
    (ALL.MOD-A (IN.P (K (PLUR RAG.N)))))))
 AND.CC
 ((ADV-A (FOR.P (AN.D ALMS.N))) (IMPORTUNE.V (EVERY.D PASSENGER.N))))
```
Predicted constituent types ((list of types) -- constituent)
  ```
((VERB PRED) --
 ((PASV FOLLOW.V)
  (BY.P-ARG
   (NP+PREDS ((THREE.D FOUR.D OR.CC SIX.D) (PLUR CHILD.N))
    (ALL.MOD-A (IN.P (K (PLUR RAG.N))))))))
```
  ```
((UNKNOWN) -- AND.CC)
```
  ```
((VERB PRED) --
 ((ADV-A (FOR.P (AN.D ALMS.N))) (IMPORTUNE.V (EVERY.D PASSENGER.N))))
```

**Comment by genelkim** [comment]
************************************
## Sanity checking formula (before preprocessing).
```
(AND.ADV-S
 ((AS.PS
   ((ADV-E (AT.P (THIS.D MOMENT.N)))
    ((THE.D ORCHESTRA.N)
     ((PAST GIVE.V)
      (THE.D (N+PREDS SIGNAL.N (FOR.P (THE.D WALTZ.N))))))))
  (|Albert|
   (((PAST PUT.V) (HIS.D ARM.N)
     (ROUND.P (THE.D (N+PREDS WAIST.N (OF.P (THE.D COUNTESS.N))))))
    AND.CC
    ((PAST DISAPPEAR.V) (ADV-A (WITH.P HER.PRO))
     (ADV-A
      (IN.P (THE.D (N+PREDS WHIRL.N (OF.P (K (PLUR DANCER.N))))))))))))
```

## Sanity checking formula (after preprocessing).
```
((AND.ADV-S
  ((AS.PS
    ((ADV-E (AT.P (THIS.D MOMENT.N)))
     ((THE.D ORCHESTRA.N)
      ((PAST GIVE.V)
       (THE.D (N+PREDS SIGNAL.N (FOR.P (THE.D WALTZ.N))))))))
   (|Albert|
    (((PAST PUT.V) (HIS.D ARM.N)
      (ROUND.P (THE.D (N+PREDS WAIST.N (OF.P (THE.D COUNTESS.N))))))
     AND.CC
     ((ADV-A
       (IN.P (THE.D (N+PREDS WHIRL.N (OF.P (K (PLUR DANCER.N)))))))
      ((PAST DISAPPEAR.V) (ADV-A (WITH.P HER.PRO))))))))
 NIL NIL)
```

## Possible errors

Possibly failed conditions:
  _"Verbs (both tensed and untensed) *must* take all non-subject arguments in a flat construction."_
Ann segment:
  ```
(((PAST PUT.V) (HIS.D ARM.N)
  (ROUND.P (THE.D (N+PREDS WAIST.N (OF.P (THE.D COUNTESS.N))))))
 AND.CC
 ((ADV-A (IN.P (THE.D (N+PREDS WHIRL.N (OF.P (K (PLUR DANCER.N)))))))
  ((PAST DISAPPEAR.V) (ADV-A (WITH.P HER.PRO)))))
```
Predicted constituent types ((list of types) -- constituent)
  ```
((PRED TENSED-VERB) --
 ((PAST PUT.V) (HIS.D ARM.N)
  (ROUND.P (THE.D (N+PREDS WAIST.N (OF.P (THE.D COUNTESS.N)))))))
```
  ```
((UNKNOWN) -- AND.CC)
```
  ```
((PRED TENSED-VERB) --
 ((ADV-A (IN.P (THE.D (N+PREDS WHIRL.N (OF.P (K (PLUR DANCER.N)))))))
  ((PAST DISAPPEAR.V) (ADV-A (WITH.P HER.PRO)))))
```

## Formula with predicted types (for debugging).

```
(((TYPES TENSED-SENT)
  (AND.ADV-S
   ((TYPES TENSED-SENT)
    (((TYPES SENT-MOD)
      (AS.PS
       ((TYPES TENSED-SENT)
        (((TYPES ADV-E SENT-MOD)
          (ADV-E
           ((TYPES PP PRED) (AT.P ((TYPES TERM) (THIS.D MOMENT.N))))))
         ((TYPES TENSED-SENT)
          (((TYPES TERM) (THE.D ORCHESTRA.N))
           ((TYPES PRED TENSED-VERB)
            (((TYPES PRED TENSED-VERB) (PAST GIVE.V))
             ((TYPES TERM)
              (THE.D
               ((TYPES NOUN PRED)
                (N+PREDS SIGNAL.N
                 ((TYPES PP PRED)
                  (FOR.P ((TYPES TERM) (THE.D WALTZ.N))))))))))))))))
     ((TYPES TENSED-SENT)
      (|Albert|
       ((TYPES PRED TENSED-VERB)
        (((TYPES PRED TENSED-VERB)
          (((TYPES PRED TENSED-VERB) (PAST PUT.V))
           ((TYPES TERM) (HIS.D ARM.N))
           ((TYPES PP PRED)
            (ROUND.P
             ((TYPES TERM)
              (THE.D
               ((TYPES NOUN PRED)
                (N+PREDS WAIST.N
                 ((TYPES PP PRED)
                  (OF.P ((TYPES TERM) (THE.D COUNTESS.N))))))))))))
         AND.CC
         ((TYPES PRED TENSED-VERB)
          (((TYPES ADV-A)
            (ADV-A
             ((TYPES PP PRED)
              (IN.P
               ((TYPES TERM)
                (THE.D
                 ((TYPES NOUN PRED)
                  (N+PREDS WHIRL.N
                   ((TYPES PP PRED)
                    (OF.P
                     ((TYPES TERM)
                      (K
                       ((TYPES NOUN PRED) (PLUR DANCER.N))))))))))))))
           ((TYPES PRED TENSED-VERB)
            (((TYPES PRED TENSED-VERB) (PAST DISAPPEAR.V))
             ((TYPES ADV-A)
              (ADV-A ((TYPES PP PRED) (WITH.P HER.PRO)))))))))))))))))
```

NIL

####################################

**Comment by genelkim** [comment]
Fixed in faa6a03fe73c3b9feab589b03c2bf0ac7f6a9adb -- updated at next production update
